pkg/doc/verifiable/credential_jwt_test.go

```diff
@@ -10,12 +10,15 @@ import (
 	"testing"
 	"time"
 
-	"github.com/square/go-jose/v3/jwt"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/jwt"
```

Comment:
double check import order .. usually Aries-framework-go should be the last import line